.github/workflows/indent.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         cmake -DDEAL_II_COMPONENT_DOCUMENTATION=ON -DDEAL_II_DOXYGEN_USE_MATHJAX=ON ..
         make -j 2 documentation
         cat doxygen.log
+        sed -i '/Inheritance graph/d' doxygen.log
```

Comment:
Could you add a comment explaining which warning this suppresses?

Reply:
Done.